poi-ooxml/src/test/java9/module-info.java

```diff
@@ -81,6 +81,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
     exports org.apache.poi.poifs.crypt.temp;
 
     opens org.apache.poi.openxml4j.opc to org.apache.poi.poi, org.junit.platform.commons;
+    opens org.apache.poi.openxml4j to org.apache.poi.ooxml, org.junit.platform.commons;
```

Comment:
Don't know the structure of module-info, but could this line be merged with the previous one by adding `org.apache.poi.ooxml` there?